libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergContainerFragment.java

```diff
@@ -311,11 +311,11 @@ public void onRedoPressed() {
         mWPAndroidGlueCode.onRedoPressed();
     }
 
-    public void updateCapabilities(GutenbergPropsBuilder gutenbergPropsBuilder) {
+    public void updateCapabilities(@Nullable GutenbergPropsBuilder gutenbergPropsBuilder) {
         // We want to make sure that activity isn't null
         // as it can make this crash to happen: https://github.com/wordpress-mobile/WordPress-Android/issues/13248
         final Activity activity = getActivity();
-        if (activity != null) {
+        if (activity != null && gutenbergPropsBuilder != null) {
```

Comment:
Since `updateCapabilities()` doesn't do anything if `gutenbergPropsBuilder` is null, I think it shouldn't accept null `gutenbergPropsBuilder`. And we should handle nullability where we are calling `updateCapabilities()` based on the case. WDYT? I wonder in which cases the db is null and what we're doing for these cases in other places of the app.

Reply:
Agreed, handling nullability in the function calling `updateCapabilities()` does make more sense to me. I've updated the PR if you'd like to take a look again, appreciate any other feedback!